.github/workflows/go.yml

```diff
@@ -30,28 +30,8 @@ jobs:
       env:
           GOOS: linux
           GOARCH: amd64
-    - name: Build kubectl-tbac Mac binary
```

Comment:
Don't you need to build the binaries still?

Reply:
wops, not sure how I got this commit into the branch.. Will remove it.